gimel/gimel.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import print_function
 import sys
 sys.path.insert(0, './vendor')
-import redis
-from config import config
+import redis  # noqa
+from config import config  # noqa
```

Comment:
Solved it now with https://github.com/Alephbet/gimel/pull/5/commits/81d9db34c9484cc04d63aa66b3e28d1ecb3716c0